docs/sources/operations/zone-ingesters.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Zone Aware Ingesters
+description: Migration guide and additional details
+weight: 10
+---
+
+# Zone Aware Ingesters
+Loki's zone aware ingesters are used by Grafana Labs in order to allow for easier rollouts of large Loki deployments. You can think of them as three logical zones, however with some extra k8s config you could deploy them in separate zones.
+
+By default, an incomming log stream's logs are replicated to 3 random (but the same 3, except in the case of some replica scaling) ingesters. This means that if one ingester is restarted, no data is lost, but two can mean data is lost and also impacts the systems ability to ingest logs because of an unhealthy ring status.
```

Comment:
Would saying "three randomly assigned ingesters" capture what you're saying in parentheses?  Also I feel like there's a word missing in the second sentence?

Reply:
Here's what I was trying to say, in too many words, open to suggestions how to phrase it differently.

> By default, an incoming log stream's logs are replicated to 3 random ingesters. Except in the case of some replica scaling up or down, a given stream will always be replicated to the same 3 ingesters. This means that if one of those ingesters is restarted no data is lost, but two restarting can mean data is lost and also impacts the systems ability to ingest logs because of an unhealthy ring status.

Reply:
Your phrasing in the comment is clearer (at least to me) than the original text.  I'd go with the text from the comment.